benchmarks/kn-performance/gradle.properties

```diff
@@ -0,0 +1,17 @@
+compose.version=1.5.1
+kotlin.version=1.9.10
+agp.version=7.0.4
+org.gradle.jvmargs=-Xmx3g
+kotlin.code.style=official
+kotlin.native.cacheKind=none
```

Comment:
remove this

Reply:
The following properties are mot required as well:

kotlin.native.enableDependencyPropagation=false
kotlin.mpp.enableGranularSourceSetsMetadata=true
# Enable kotlin/native experimental memory model
kotlin.native.binary.memoryModel=experimental

Reply:
```
Compilation failed: Java heap space

 * Source files: 
 * Compiler version: 1.9.10
 * Output kind: PROGRAM
 ```
 When `org.gradle.jvmargs=-Xmx3g` is removed.

Reply:
Removed others